collectors/crowdstrike/local/sam-template.yaml

```diff
@@ -0,0 +1,36 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Description: Run it locally
+Resources:
+  LocalLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      KmsKeyArn: your-arn-here
+      Environment:
+        Variables:
+          AWS_LAMBDA_FUNCTION_NAME:
+          aims_secret_key:
+          aims_access_key_id:
+          al_api:
+          stack_name:
+          azcollect_api:
+          ingest_api:
+          DEBUG:
+          paws_state_queue_arn:
+          paws_state_queue_url:
+          paws_poll_interval:
+          paws_extension:
+          collector_id:
+          paws_ddb_table_name:
+          paws_max_pages_per_invocation:
+          paws_secret_param_name:
+          paws_api_client_id:
+          paws_endpoint:
+          collector_streams:
+          customer_id:
+      CodeUri: /Users/sameerbagade/alert-logic/paws-collector/collectors/crowdstrike
+      Runtime: nodejs10.x
```

Comment:
could you please bump it to 12 here and in the [template](https://github.com/alertlogic/paws-collector/blob/master/collectors/template/local/sam-template.yaml#L25)

Reply:
Done